app/packages/core/src/plugins/SchemaIO/utils/index.ts

```diff
@@ -92,6 +92,27 @@ const COMPOSITE_VIEWS = [
   "ButtonGroupView",
 ];
 
+const NON_EDITABLE_VIEWS = [
```

Comment:
This seems hard to maintain. I would rather we define this in each view somehow or in a way that we won't forget to add things to this list. In other words I don't think we'll be able to maintain this list.

Reply:
Yah, good point. I will follow-up. I would say let's go with this for now to unblock Minh's panel for release